frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/SignUpHandler.java

```diff
@@ -105,28 +106,23 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
         Optional<ErrorResponse> passwordValidationError =
                 passwordValidator.validate(request.getPassword());
 
-        var restrictedSection =
-                new AuditService.RestrictedSection(
-                        Optional.ofNullable(userContext.getTxmaAuditEncoded()));
-
-        if (passwordValidationError.isEmpty()) {
-            LOG.info("No password validation errors found");
-            if (authenticationService.userExists(request.getEmail())) {
-
-                auditService.submitAuditEvent(
-                        FrontendAuditableEvent.CREATE_ACCOUNT_EMAIL_ALREADY_EXISTS,
-                        userContext
-                                .getClient()
-                                .map(ClientRegistry::getClientID)
-                                .orElse(AuditService.UNKNOWN),
+        var auditContext =
+                new AuditContext(
+                        userContext.getClientId(),
                         userContext.getClientSessionId(),
                         userContext.getSession().getSessionId(),
                         AuditService.UNKNOWN,
                         request.getEmail(),
                         IpAddressHelper.extractIpAddress(input),
                         AuditService.UNKNOWN,
                         PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders()),
-                        restrictedSection);
+                        Optional.ofNullable(userContext.getTxmaAuditEncoded()));
+
+        if (passwordValidationError.isEmpty()) {
+            LOG.info("No password validation errors found");
```

Comment:
Is this going to be useful in prod?

Reply:
You're probably right that it's not, but I didn't add this line here - github diff here not being particularly smart. So I'd propose not to touch it, but I think maybe we as a team should discuss logging strategy.